i18n/README.zh-hk.md

```diff
@@ -59,6 +60,16 @@ C++從頭開始構建的，沒有任何外部依賴，並且 100% 不受垃圾
 - 一個即時的加密貨幣（比特幣、乙太幣）交易數據集。
 - 一個包括 25 萬艘船的時序地理數據集。
 
+| 查詢                                                                          | 運行時間                                                                                                                                                                         |
+| ----------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
same as on English version. The queries are not using the real column names, but generic data type names. Then on the link they use the real query. In some of the queries we are also showing a different query (not just the column names) on the table and on the link. This is very confusing for me